addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -2046,7 +2040,26 @@ MockServer.include({
         return false;
     },
     /**
-     * Simulates `get_display_value` on `mail.tracking.value`
+     * Simulates `_tracking_value_format` on `mail.tracking.value`
+     */
+    _mockMailTrackingValue_TrackingValueFormat(tracking_value_ids) {
+        const tracking_values = tracking_value_ids.map(tracking => ({
+            changedField: tracking.field_desc,
+                id: tracking.id,
+                newValue: [['insert-and-replace', {
+                    fieldType: tracking.field_type,
+                    value: this._mockMailTrackingValue_GetDisplayValue(tracking, 'new')}]],
+                oldValue: [['insert-and-replace', {
+                    fieldType: tracking.field_type,
+                    value: this._mockMailTrackingValue_GetDisplayValue(tracking, 'old')}]],
```

Comment:
Isn't the indentation a bit weird here?

Reply:
cause it is too long to put into one line, so indention is needed for sure.
for me, this is how i deal with the structure most usual. is there a better way to do the indentions here? any advice?

Reply:
I mean, shouldn't the `id` key (for instance) be aligned to the same column as `changedField`?

Reply:
> I mean, shouldn't the `id` key (for instance) be aligned to the same column as `changedField`?

Indeed. `changedField`, `id`, `newValue`, `oldValue` should all be at the same indent level.